contrib/internal/httptrace/options.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package httptrace
+
+import "os"
+
+type config struct {
+	ipHeader string
+}
+
+var (
+	clientIPHeaderEnvVar = "DD_CLIENT_IP_HEADER"
+	cfg                  = newConfig()
+)
+
+func newConfig() *config {
```

Comment:
Why not just:
```go
var storeClientIP = os.Getenv("DD_TRACE_HTTP_CLIENT_IP_HEADER") != ""
```
...and dump everything else?

Reply:
It's likely that some more env vars will be added to configure `httptrace`, which would make for a "bigger" configuration. The first that comes to mind is `DD_TRACE_HEADER_TAGS`, although we don't plan on handling the headers tags right away. Anyway, we can drop this for now and address this later when needed. Sounds good?

Reply:
Yes, please. Let's not prematurely add things to prepare for some future requirement.

Reply:
Also, given that the default setting is a bunch of headers, won't users want to specify a list here instead of just one header?

Reply:
It's been agreed upon that a user should only specify a single header and that the presence of multiple header is a sign of misconfiguration or malicious request. The default behaviour is to check if one and only one header from the default list is present. If the env var is used however, that one and only header to check is specified using said env var.